.github/workflows/ci_cd.yml

```diff
@@ -175,3 +183,23 @@ jobs:
           docker tag ${{ matrix.image }} \
             ghcr.io/wordpress/openverse-${{ matrix.image }}:${{ github.ref_name }}
           docker push --all-tags ghcr.io/wordpress/openverse-${{ matrix.image }}
+
+  docs:
+    name: Publish developer docs
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && github.repository == 'WordPress/openverse-api'
+    needs:
+      - test-api
+    steps:
+      - name: Download developer docs
+        uses: actions/download-artifact@v2
+        with:
+          name: docs
+          path: /tmp/docs
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@bbdfb200618d235585ad98e965f4aafc39b4c501 # v3.7.3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: /tmp/docs
+          force_orphan: true
```

Comment:
I've seen this done for GitHub pages elsewhere, do you know why this needs to be orphaned? This matter very little but I don't love seeing the little orphaned commits in the git history graph :sweat_smile:

Reply:
I suppose it's because tracking Git history for autogenerated files is redundant but we have to, because GitHub wouldn't host files if they weren't a part of the VCS? I hope [the documentation](https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-force-orphan-force_orphan) for this feature can help.